File: MPT.CSI/API/Core/Program/ModelBehavior/Definition/eLoadPatternType.cs

namespace MPT.CSI.API.Core.Program.ModelBehavior.Definition
{
    /// <summary>
    /// Load pattern types available in the application.
    /// </summary>
    public enum eLoadPatternType
    {
        None = 0,
        Dead = 1,
        SuperDead,
        Live,
        LiveReducible,
        Quake,
        Wind,
        Snow,
        Other,
        Move,
        Temperature,
        LiveRoof,
        Notional,
        LivePattern,
        Wave,
        Braking,
        Centrifugal,
        Friction,
        Ice,
        WindOnLiveLoad,
        EarthPressureHorizontal,
        EarthPressureVertical,
        EarthSurcharge,
        DownDrag,
        VehicleCollision,
        VesselCollision,
        TemperatureGradient,
        Settlement,
        Shrinkage,
        Creep,
        WaterLoadPressure,
        LiveLoadSurcharge,
        LockedInForces,
        LiveLoadPedestrian,
        PreStress,
        HyperStatic,
        Bouyancy,
        StreamFlow,
        Impact,
        Construction
    }
}

File: MPT.CSI/OOAPI/MPT.CSI.OOAPI/Old VB Files/API Library/Enumerations/CSiConversions/eLoadPatternType.vb
Option Explicit On
Option Strict On

Public Enum eLoadPatternType
    None = 0
    Dead = 1
    SuperDead
    Live
    LiveReducible
    Quake
    Wind
    Snow
    Other
    Move
    Temperature
    LiveRoof
    Notional
    LivePattern
    Wave
    Braking
    Centrifugal
    Friction
    Ice
    WindOnLiveLoad
    EarthPressureHorizontal
    EarthPressureVertical
    EarthSurcharge
    DownDrag
    VehicleCollision
    VesselCollision
    TemperatureGradient
    Settlement
    Shrinkage
    Creep
    WaterLoadPressure
    LiveLoadSurcharge
    LockedInForces
    LiveLoadPedestrian
    PreStress
    HyperStatic
    Bouyancy
    StreamFlow
    Impact
    Construction
End Enum
